src/components/organisms/NavBar/index.js

```diff
@@ -67,33 +64,51 @@ const NavBar = ({
               width={195}
             />
           </Box>
-          {linksSorted.map((link, index) => {
-            const bpIndex = breakpoints.indexOf(link.breakpoint)
-            const display =
-              bpIndex !== -1
-                ? Array(bpIndex + 2)
-                    .fill("none")
-                    .fill("block", bpIndex + 1)
-                : "block"
-            return (
-              <React.Fragment key={link.href}>
-                {index === splitIndex && <Box ml="auto" key="separator" />}
+          <Flex
+            sx={{
+              flexDirection: "row",
+              justifyContent: primaryLink
+                ? "flex-start"
+                : ["flex-end", "flex-end", "flex-end", "flex-start"],
```

Comment:
I would suggest to remove those ternaries from the props and to place it before the render function, in order to make the code more clear. If you place it in that `links` `useMemo` callback it would be even better, because the ternary wouldn't be performed on each render, right?

```
const defaultFlexJustifyContent = [ "flex-end", "flex-end", "flex-end", "flex-start" ]
const primaryNavBarButtonDisplay = [ "none", "none", "block" ]
const [ flexJustifyContent, navBarButtonDisplay ] = primaryLink
                  ? [ "flex-start",  primaryNavBarButtonDisplay ]
                  : [ defaultFlexJustifyContent, "block" ]
```
```suggestion
              justifyContent: {flexJustifyContent},
```

Reply:
I'm not sure if that makes the code clearer. If anything it introduces the gap between declaration and application, which doesn't work at all for CSS. The only real reason to alias CSS properties via variables is for DRY, which is not the case here.

Ternaries are VERY fast, there's no real reason to optimize them.